packages/sku/config/eslint/ignores.js

```diff
@@ -0,0 +1,40 @@
+// @ts-check
+
+/**
+ * Create a new ESLint ignores configuration object
+ *
+ * @param {object} options
+ * @param {boolean} options.hasLanguagesConfig - Whether 'languages' is configured in sku config
+ * @param {string | undefined} options.target - The configured target directory in sku config
+ */
+const createEslintIgnoresConfig = ({ hasLanguagesConfig, target }) => {
+  // ESLint migrates ignore entries differently depending on whether a path contains a subdirectory.
+  // We want the sku target to match the migrated entry so it ends up excluded in the migrated
+  // output.
+  const targetIgnore =
+    target && target.includes('/') ? `${target}/` : `**/${target}/`;
+
+  const ignores =
```

Comment:
**thought:** Worth mentioning the ignore list in the docs?

Reply:
Yeah I think so.

Reply:
Added in https://github.com/seek-oss/sku/pull/1094/commits/9c1fd38cca36e8e8d06d45917bdd7405e0db02d6. Also realized I didn't add an entry for `eslintIgnore` to the configuration page, so did that too.